### ISSUE TITLE
If i cover my light sensor, that auto adjusts the brightness of my phone, the whole screen goes black

### ISSUE THREAD

**Issue by CodeCubeNeo** [vector-im/element-android#5431]
### Steps to reproduce

If i am in a middle of a call (voice or video) and I cover the front sensor that ig receives light and is responsible for auto adjusting the phone brightness, so that it doesnt receive any light, the whole screen goes black. Nothing helps, even if i click the power button. Only If i uncover the sensor the screen lights back up. It may have something to do that Element requires some proof, that someone is in front of the phone or i don't know.

### Outcome

If i cover the sensor nothing should happen. I think it is not phone/hardware related, since with other messaging apps, like facebook messenger, nothing happens if i cover my light sensor, but here the screen goes black.

### Your phone model

Huawei Mate 9 

### Operating system version

Android 9

### Application version and app store

Element version 1.4.2., olm version 3.2.10

### Homeserver

matrix.org

### Will you send logs?

Yes

**Comment by ouchadam** [comment]
If I've understood correctly, this is intentional for voice calls and is common practice for telephone apps. The feature is used to prevent accidental screen presses whilst the device is next to your ear 

however the power button not working does sound odd...

**Comment by CodeCubeNeo** [comment]
Yes. I guess that that the screen goes blank is normal, but the power button should bring that screen back up again (just like in normal android calls).

**Comment by CodeCubeNeo** [comment]
Any progress on this?

**Comment by CodeCubeNeo** [comment]
Any progress on this?
